assets/scss/cards.scss

```diff
@@ -1,3 +1,94 @@
+.content-card {
+  background: $extralightgrey;
+  border-radius: $border-radius-small;
+  box-shadow: $boxshadow-small;
+  color: $black;
+  font-size: $font-size-extrasmall;
+  height: auto;
+  line-height: 1.1875rem;
+  transition: box-shadow 0.25s;
+
+  @media (min-width: $bp-medium) {
+    min-height: 20rem;
+  }
+
+  &:hover {
+    box-shadow: $boxshadow-large;
+    background-color: $lightgrey;
+  }
+
+  a {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .card-img {
+    background-position: center center;
+    background-size: cover;
+    border-top-left-radius: 4px;
+    border-top-right-radius: 4px;
```

Comment:
Could we use shorthand here? `border-radius: 4px 4px 0 0`

Reply:
Good idea, I have updated it to a shorthand version.